guides-markdown/multisig-wallet.md

```diff
@@ -0,0 +1,370 @@
+# Using bcoin wallet node for multisig wallet
+
+```post-author
+Joe Miyamoto <joemphilips@gmail.com>
+```
+
+```post-description
+How to use bcoin with a multisig wallet and account. For both as a hot-wallet and as a watch-only-wallet.
+```
+
+## How it works
+
+Bitcoin multisig account is an account which holds more than 1 public key,
+and generates P2SH or P2WSH address for those public keys.
+When you want to use those funds,
+you must create a multisig transaction which has been signed by private keys corresponding to those public keys.
+See [our multisig transaction guide](http://bcoin.io/guides/multisig-tx.html) for more detail about the multisig transaction itself.
+
+In this guide, we take it a little step further and see a whole lifecycle of the multisig account in a more practical way.
+
+## Usecases
+
+### 1. Personal wallet which is secure against key compromise
+
+Usually in a cryptocurrency world, when you lose your master key and it's seed for the wallet there is no way to recover your funds.
+But by splitting your secret keys into ...
+
+1. your own laptop.
+2. hardware wallet you own.
+3. a paper wallet in your personal vault
+
+...your funds will be safe unless more than one key is compromised at the same time.
+In this case, you may use 1. and 2. for usual spending,
+and only need 3 if you lose your secret keys 1 and 2.
+
+### 2. Wallet held by more than one person
+
+Sometimes it is desireable to distribute the control of funds among several people.
+For example, funds for a corporation or for a crowdfunding project.
+We can mitigate the risk of fund loss by utilizing a multisig in cases where single person with control:
+
+1. Is no longer in charge
```

Comment:
Not super important, but more of a nit, in markdown you can put all of your numbers as 1 and then they will render 1,2,3,4,etc - it helps if you want to quickly reorder things at some point in the future